packages/block-editor/src/components/iframe/content.scss

```diff
@@ -1,5 +1,6 @@
 .block-editor-iframe__body {
 	position: relative;
+	border: 0.01px solid transparent;
```

Comment:
A comment here would help the readability here I believe 👍

Reply:
Wondering if this should be `outline` rather than border, unless the inset style is preferred?

| Border  (trunk) | Outline |
|--------|--------|
| <img width="273" alt="Screenshot 2024-10-23 at 10 49 24 am" src="https://github.com/user-attachments/assets/93c548fb-3db3-4bc5-b393-83d68541dc84"> | <img width="277" alt="Screenshot 2024-10-23 at 10 49 18 am" src="https://github.com/user-attachments/assets/2501f6f0-7315-4ea6-857f-783c00f7c12c"> |

Reply:
Actually, ignore me please - refreshing and updating trunk helps 😄

Reply:
I'd still use outline - the border is causing some minor layout discrepancies:


https://github.com/user-attachments/assets/f5b2dd18-4ca8-40fc-b6c2-1929e5ebbe79

Reply:
I don’t think outline can work. The border is to cause margins of children to be contained something outline won’t do. That is this issue will return:

https://github.com/user-attachments/assets/27fff5a9-92fd-40df-bd8d-1b0a92cca820

Reply:
> The border is to cause margins of children to be contained something outline won’t do.

Ah gotcha, thanks for the explainer.

I was testing in the site editor where the effect wasn't so pronounced. 

I wonder if there's another property to prevent margin collapsing, e.g, `display: flow-root;`

Reply:
`display: flow-root` is looking good from my testing.

Reply:
`box-shadow` is also an alternative to display a visual border without causing layout shifts

Reply:
I think the goal was to prevent margin collapsing rather than displaying a border. The border in this PR is transparent.

See related: 

- https://github.com/WordPress/gutenberg/pull/66390